<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
selection-background-color: rgb(245, 121, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>511</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 121, 0);
font: 75 italic 16pt &quot;EB Garamond SC&quot;;</string>
    </property>
    <property name="text">
     <string>Witaj w aplikacji QtFinance!</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>530</y>
      <width>508</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of plot&lt;br/&gt;in document:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbWidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>480</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbHeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>340</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUseCurrentSize">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);
border-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Use current</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);</string>
         </property>
         <property name="text">
          <string>MAKE</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>418</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>573</y>
      <width>562</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionArtyku_y_spo_ywcze"/>
   <addaction name="actionKosmetyki"/>
   <addaction name="actionOdzie"/>
   <addaction name="actionRachunki"/>
   <addaction name="actionRozrywka_i_wypoczynek"/>
   <addaction name="actionRozrywka"/>
   <addaction name="actionTransport"/>
   <addaction name="actionUroda"/>
   <addaction name="actionZdrowie"/>
   <addaction name="actionBez_kategorii"/>
   <addaction name="separator"/>
   <addaction name="actionDoch_d"/>
   <addaction name="actionDane"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWydatki">
    <property name="title">
     <string>Wydatki</string>
    </property>
    <addaction name="actionArtyku_y_spo_ywcze"/>
    <addaction name="actionKosmetyki"/>
    <addaction name="actionOdzie"/>
    <addaction name="actionRachunki"/>
    <addaction name="actionRozrywka_i_wypoczynek"/>
    <addaction name="actionRozrywka"/>
    <addaction name="actionTransport"/>
    <addaction name="actionUroda"/>
    <addaction name="actionZdrowie"/>
    <addaction name="separator"/>
    <addaction name="actionBez_kategorii"/>
   </widget>
   <widget class="QMenu" name="menuPrzychody">
    <property name="title">
     <string>Przychody</string>
    </property>
    <addaction name="actionDoch_d"/>
   </widget>
   <widget class="QMenu" name="menuAnaliza_finansowa">
    <property name="title">
     <string>Analiza finansowa</string>
    </property>
    <addaction name="actionDane"/>
   </widget>
   <addaction name="menuWydatki"/>
   <addaction name="menuPrzychody"/>
   <addaction name="menuAnaliza_finansowa"/>
  </widget>
  <action name="actionArtyku_y_spo_ywcze">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-hamburger-80.png</normaloff>:/images/icons-hamburger-80.png</iconset>
   </property>
   <property name="text">
    <string>Artykuły spożywcze</string>
   </property>
  </action>
  <action name="actionUroda">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-barbershop-80.png</normaloff>:/images/icons-barbershop-80.png</iconset>
   </property>
   <property name="text">
    <string>Uroda</string>
   </property>
  </action>
  <action name="actionKosmetyki">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons8-tube-80.png</normaloff>:/images/icons8-tube-80.png</iconset>
   </property>
   <property name="text">
    <string>Drogeria</string>
   </property>
  </action>
  <action name="actionRachunki">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-coins-80.png</normaloff>:/images/icons-coins-80.png</iconset>
   </property>
   <property name="text">
    <string>Rachunki</string>
   </property>
  </action>
  <action name="actionRozrywka_i_wypoczynek">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons8-food-and-wine-96.png</normaloff>:/images/icons8-food-and-wine-96.png</iconset>
   </property>
   <property name="text">
    <string>Restauracje</string>
   </property>
  </action>
  <action name="actionBez_kategorii">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-folder-80.png</normaloff>:/images/icons-folder-80.png</iconset>
   </property>
   <property name="text">
    <string>Bez kategorii</string>
   </property>
  </action>
  <action name="actionOdzie">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-t-shirt-80.png</normaloff>:/images/icons-t-shirt-80.png</iconset>
   </property>
   <property name="text">
    <string>Moda</string>
   </property>
  </action>
  <action name="actionTransport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-airplane-80.png</normaloff>:/images/icons-airplane-80.png</iconset>
   </property>
   <property name="text">
    <string>Transport</string>
   </property>
  </action>
  <action name="actionZdrowie">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-heart-with-pulse-80.png</normaloff>:/images/icons-heart-with-pulse-80.png</iconset>
   </property>
   <property name="text">
    <string>Zdrowie</string>
   </property>
  </action>
  <action name="actionDoch_d">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons-increase-80.png</normaloff>:/images/icons-increase-80.png</iconset>
   </property>
   <property name="text">
    <string>Dochód</string>
   </property>
  </action>
  <action name="actionDane">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons8-futures-80.png</normaloff>:/images/icons8-futures-80.png</iconset>
   </property>
   <property name="text">
    <string>Dane</string>
   </property>
  </action>
  <action name="actionRozrywka">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons8-stormtrooper-80.png</normaloff>:/images/icons8-stormtrooper-80.png</iconset>
   </property>
   <property name="text">
    <string>Rozrywka</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
